docs/user/InstructionsForAutoTuner.md

```diff
@@ -0,0 +1,201 @@
+
+AutoTuner provides two main functionalities as follows.
+* Automatic hyperparameter tuning framework for OpenROAD-flow-script (ORFS)
+* Parametric sweeping experiments for ORFS
+
+
+AutoTuner contains top-level Python script for ORFS, each of which implements a different search algorithm. Current supported search algorithms are as follows.
+* Random/Grid Search 
+* Population Based Training ([PBT](https://deepmind.com/blog/article/population-based-training-neural-networks))
+* Tree Parzen Estimator ([HyperOpt](http://hyperopt.github.io/hyperopt))
+* Bayesian + Multi-Armed Bandit ([AxSearch](https://ax.dev/))
+* Tree Parzen Estimator + Covariance Matrix Adaptation Evolution Strategy ([Optuna](https://optuna.org/))
+* Evolutionary Algorithm ([Nevergrad](https://github.com/facebookresearch/nevergrad))
+
+User-settable coefficient values (`coeff_perform`, `coeff_power`, `coeff_area`) of three objectives to set the direction of tuning are written in the script. Each coefficient is expressed as a global variable at the `get_ppa` function in `PPAImprov` class in the script (`coeff_perform`, `coeff_power`, `coeff_area`). Efforts to optimize each of the objectives are proportional to the specified coefficients.
+
+
+## Input JSON structure
+
+Sample JSON file for sky130hd aes design: [[link]](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/sky130hd/aes/autotuner.json)
+
+Simple Example with comments:
+```json
+{
+    "_SDC_FILE_PATH": "constraint.sdc",     // pointer of the base SDC file for modification
+    "_SDC_CLK_PERIOD": {                    // parameter name (clock period) for sweeping/tuning. 
+        "type": "float",                    // parameter type for sweeping/tuning
+        "minmax": [                         // min-to-max range for sweeping/tuning. 
+            1.0,                            // The unit follows the default value of each technology std cell library.
+            3.7439
+        ],
+        "step": 0                           // step ‘0’ for type ‘float’ means continuous step for sweeping/tuning
+    },
+    "CORE_MARGIN": {                        // parameter name (die-to-core margin) for sweeping/tuning
+        "type": "int",
+        "minmax": [
+            2,
+            2
+        ],
+        "step": 0                           // step ‘0’ for type ‘int’ means the constant parameter.
+    },
+}
+```
+
+## Tunable / sweepable parameters
+
+Tables of parameters that can be swept/tuned in technology platforms supported by ORFS.
+Any variable that can be set from the command line can be used for tune or sweep.
+
+For SDC you can use:
+
+* `_SDC_FILE_PATH`
+  - Path relative to the current JSON file to the SDC file.
+* `_SDC_CLK_PERIOD`
+  - Design clock period. This will create a copy of _SDC_FILE_PATH and modify the clock period.
```

Comment:
Wrap all underscore variables with a backtick to ensure they will be correctly rendered.